packages/spark-core/components/_promo.scss

```diff
@@ -0,0 +1,36 @@
+// ==================================================================
+// Promo
+// ==================================================================
+.sprk-c-Promo {
+  max-width: $promo-max-width;
+  margin: 0 auto;
+  border: $promo-border;
+}
+
+.sprk-c-Promo__title {
+  font-weight: $font-weight-display-four;
```

Comment:
For this, if the desire is to have the Promo title always be font-weight: normal; then we should set it as that and not use the $font-weight-display-four var. New type settings will change that value to bold and then our intent for this pattern is lost.